azure-pipelines-deploy-ni-redirects.yml

```diff
@@ -34,6 +43,35 @@ jobs:
           steps:
             - download: none
             - checkout: self
+            - script: |
+                sudo apt-get update
+                sudo apt-get install -y jq
```

Comment:
If we need these tools ideally we get them built into the agent image, but we can handle that after we know its working.

Reply:
Sounds good, happy to remove stage when we add to image